### PR TITLE
SEO-171754- xamarin controls

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @Aishwarya-Ganesan ,

Have made the changes in redirect links. Please review.

Thanks.
Regards,
Edith.

### CODE REVIEW

File: Xamarin/DataGrid/Getting-started.md [hunk 1]
```diff
@@ -493,9 +493,9 @@ The selection operations can be handled with the help of the [SelectionChanging]
...
+The StackLayout positions the child element one after the other. They are adding either horizontally or vertically. Space of the [StackLayout](https://learn.microsoft.com/en-us/dotnet/api/Xamarin.Forms.StackLayout?view=xamarin-forms) depends on the HorizontalOptions and VerticalOptions properties are set. But by default, the `StackLayout` will try to use the entire screen.
 
-The SfDataGrid control can be loaded inside any layout such as [Grid](https://docs.microsoft.com/en-us/dotnet/api/Xamarin.Forms.Grid?view=xamarin-forms), `StackLayout`, etc., When loading SfDataGrid inside a `StackLayout`, set the Horizontal and/or VerticalOptions of the SfDataGrid and its parent to “LayoutOptions.FillAndExpand” based on the orientation of the container in which the SfDataGrid is loaded.
+The SfDataGrid control can be loaded inside any layout such as [Grid](https://learn.microsoft.com/en-us/dotnet/api/xamarin.forms.grid?view=xamarin-forms), `StackLayout`, etc., When loading SfDataGrid inside a `StackLayout`, set the Horizontal and/or VerticalOptions of the SfDataGrid and its parent to “LayoutOptions.FillAndExpand” based on the orientation of the container in which the SfDataGrid is loaded.
```
Comment (Yvone-Atieno) [inline Xamarin/DataGrid/Getting-started.md:498]: Recheck the above link

File: Xamarin/Popup/Accessibility.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ documentation: UG
 
 # Accessibility in Xamarin Popup (SfPopupLayout)
 
-SfPopupLayout supports built-in [AutomationId](https://docs.microsoft.com/en-gb/dotnet/api/xamarin.forms.element.automationid?view=xamarin-forms#Xamarin_Forms_Element_AutomationId) for all its inner elements. These `AutomationId` values allow the automation framework to find and interact with the inner elements when the test scripts are run. A unique `AutomationId` is maintained for each inner element by prefixing the control's `AutomationId` with the inner element's Id.
+SfPopupLayout supports built-in [AutomationId](https://learn.microsoft.com/en-gb/dotnet/api/xamarin.forms.element.automationid?view=xamarin-forms) for all its inner elements. These `AutomationId` values allow the automation framework to find and interact with the inner elements when the test scripts are run. A unique `AutomationId` is maintained for each inner element by prefixing the control's `AutomationId` with the inner element's Id.
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply Xamarin/Popup/Accessibility.md:12]: I have made the changes in redirect link.

File: Xamarin/DataGrid/Getting-started.md (hunk 1 above)
Comment (Edithabich) [reply Xamarin/DataGrid/Getting-started.md:498]: I have updated proper redirect link here

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**